tests/tck/features/match/SeekById.feature

```diff
@@ -121,13 +121,15 @@ Feature: Match seek by id
             OR false
       RETURN v.player.name AS Name
       """
-    Then the result should be, in any order:
-      | Name               |
-      | 'Paul Gasol'       |
-      | 'James Harden'     |
-      | 'Jonathon Simmons' |
-      | 'Klay Thompson'    |
-      | 'Dejounte Murray'  |
+    Then a ExecutionError should be raised at runtime: Scan vertices or edges need to specify a limit number, or limit number can not push down.
```

Comment:
`xxx or false` should be simplified to `xxx` before the planner.

Reply:
emm... maybe we should let this case execute successfully.

Reply:
emm... maybe we'd better let this case execute successfully.

Reply:
See my comments: `xxx or false should be simplified to xxx before the planner.`
If we have done the expression simplification in the validator, this query could execute successfully.

Reply:
https://github.com/vesoft-inc/nebula/issues/5067

Reply:
better to report above issue as a bug.

Reply:
ok

Reply:
> #5067

Fixed in this pr.